### ISSUE TITLE
when user reported, in backend no information

### ISSUE THREAD

**Issue by lelorelo** [coralproject/talk#1803]
#### Do you want to request a **feature** or report a **bug**?
bug

#### Intended outcome:

show reported user in the list in `Community`-Tab

#### Actual outcome:

user is not listed, no information.

![bildschirmfoto 2018-08-14 um 14 13 25](https://user-images.githubusercontent.com/7877754/44090985-496be0dc-9fcc-11e8-95e2-891505cd8386.png)

#### How to reproduce the issue:

user reports another user, then look with the admin-user in backend. there is no information an user was ever reported.

![bildschirmfoto 2018-08-14 um 14 12 37](https://user-images.githubusercontent.com/7877754/44090992-5009ac9e-9fcc-11e8-965b-88e91923f8a8.png)

![bildschirmfoto 2018-08-14 um 14 12 46](https://user-images.githubusercontent.com/7877754/44090993-50247538-9fcc-11e8-9a35-02eebc5a205f.png)

#### Version and environment

talk v4.5.1
all browser

**Comment by lelorelo** [comment]
same happens when a user reports a comment

**Comment by wyattjoh** [comment]
Hi @lelorelo!

If the user account has already been reported before, and was approved/rejected by a moderator/admin, it will not show up under the "reported usernames" again!

The same rule applies to comments, where if the comment has been approved/rejected, it will not show up under the reported queue again either. This is in an effort to reduce work on the moderators behalf, as it assumes the moderator already took the action to evaluate the content and make a decision if it violated existing community guidelines.

**Comment by hjanuschka** [comment]
issue can be closed, it was due to a miss understanding of the reporting feature.
we actually setted the user status to "APPROVED" via our `tokenNotFound` plugin. (changed that now)


the reported user only shows `SET, CHANGED` one's - we thought that it is like "a flag" on the nickname, not sort of a "request for change"


same goes for comments, we enabled the "force moderation" - which effectivly disables the reporting feature, since its approved by human.
